docs/devices_api.yml

```diff
@@ -82,7 +82,7 @@ definitions:
         type: string
         description: Attribute description.
       value:
-        type: object
+        type: string
```

Comment:
both are, but @GregorioDiStefano was having problems with 'object' and asked me to modify it.

feel free to discard this PR, or accept it as you see fit.